template-compose/domain/src/main/java/co/nimblehq/template/compose/domain/model/UserPreferences.kt

```diff
@@ -0,0 +1,6 @@
+package co.nimblehq.template.compose.domain.model
+
+data class UserPreferences(
+    val username: String,
+    val isActive: Boolean
+)
```

Comment:
```suggestion
data class UserPreferences(
    val id: String
)
```

What about this? In attempt to not make it look like sample code 🙏

Reply:
@Tuubz Since we have no plan to update it into the sample repo, I intended to demonstrate how to access another type except for the simple string. 🤔 But we could remove it too.

Reply:
Hmmmm, any reason we would not bring it into the sample repo? 🙏 

cc @nimblehq/android-chapter

Reply:
@Tuubz I just don't see the issue with doing it. 😂 Could you please help me create the issue so that I can do it further?

Reply:
I see 👍 

Anyways, I won't create a separate issue for it, since we didn't do it for other stories either ([Example](https://github.com/nimblehq/android-templates/issues/352)).

However, I will start specifying it in the description for future issues for better clarity.

Would that be OK? 🙏

Reply:
@Tuubz Thanks. That could help. 🙏🏻

Reply:
@Tuubz I updated the PR description. I'll add it to sample repos at part 2.